include/clap/ext/draft/triggers.h

```diff
@@ -0,0 +1,132 @@
+#pragma once
+
+#include "../../plugin.h"
+#include "../string-sizes.h"
+
+static CLAP_CONSTEXPR const char CLAP_EXT_TRIGGERS[] = "clap.triggers.draft/0";
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+/// @page Trigger events
+///
+/// This extension enables the plugin to provide a set of triggers to the host.
+///
+/// Some examples for triggers:
+/// - start recording into a plugin-internal audio/note buffer
+/// - trigger a retrospective audio/note looper
+/// - trigger a sample-and-hold unit (maybe even per-voice)
+/// - retrigger an envelope during voice lifetime

```

Comment:
We don't need the "re" here, some envelopes may simply be triggered using the trigger event regardless of voice lifetime.

So I'd just say "trigger an envelope"